OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -94,15 +93,25 @@ public OptimizelyDecision Decide(string key)
         {
             return Optimizely.Decide(this, key, options);
         }
- 
+
+        /// <summary>
+        /// Returns a key-map of decision results for multiple flag keys and a user context.
+        /// </summary>
+        /// <param name="keys">list of flag keys for which a decision will be made.</param>
+        /// <returns>A dictionary of all decision results, mapped by flag keys.</returns>
+        public Dictionary<string, OptimizelyDecision> DecideForKeys(List<string> keys, OptimizelyDecideOption[] options)
```

Comment:
It's weird two paramaters are different types while they are used in a similar way.
Can we make them the same type? 
- DecideForKeys(List<string> keys, List<OptimizelyDecideOption> options) or
- DecideForKeys(string[] keys, OptimizelyDecideOption[] options)